tests/osm/test_validate_osm.py

```diff
@@ -0,0 +1,506 @@
+"""Test validate_osm."""
+import pytest
+import re
+from pyprojroot import here
+import geopandas as gpd
+from folium import Map
```

Comment:
I'm not sure what your opinion on this is, however it may be useful to use `folium.Map` or a name such as `folium_map` to avoid confusion with the built in `map`.

Reply:
Done